lib/logger/lib/logger.ex

```diff
@@ -548,8 +548,8 @@ defmodule Logger do
   ## Options
 
     * `:flush` - when `true`, guarantees all messages currently sent
-      to both Logger and Erlang's `error_logger` are processed before
-      the backend is removed
+      to both Logger and OTP's [`:error_logger`](http://erlang.org/doc/man/error_logger.html)
```

Comment:
<img width="599" alt="zrzut ekranu 2017-12-20 o 13 00 53" src="https://user-images.githubusercontent.com/76071/34206217-d8857992-e585-11e7-8975-6c9ca79d99ab.png">

oops, is it expected that in IEx the url `http://erlang.org/doc/man/error_logger.html` is shown? If not, I can look into it.

Reply:
@wojtekmach it is expected, no worries.